### DISCUSSION TITLE
Can someone help me turn this into an app?

### DISCUSSION THREAD

**Discussion by hamiltonprojects** [tailwindlabs/tailwindcss#13743] [Help]
Hi guys, I have a project that I need to turn into an app, anyone interested? I can't say too much about it before the NDA is signed, but if you're interested shoot me an email at danielh@primobranding.com

**Comment by robert2001b** [comment]
Define the purpose of your app and its target audience. Decide on the features and functionalities you want to include. If you’re not a developer yourself, you might need to hire a team of developers who can work on your project. Make sure they sign an NDA if your project is confidential. Create a design for your app’s user interface. [fidelity](https://www-netbenefits.com) This includes the layout of screens and the user experience.

**Comment by hamiltonprojects** [comment]
yup, all of those things are done. Just waiting on the budget. That's
what's put things on pause for now.
